modules/apps/portal-vulcan/portal-vulcan-api/src/main/java/com/liferay/portal/vulcan/resource/OpenAPIResource.java

```diff
@@ -57,4 +61,12 @@ public default Response getOpenAPI(
 		return null;
 	}
 
+	public Map<String, Schema> getSchemas(Set<Class<?>> resourceClasses)
```

Comment:
I find this signature really hard to understand...

Is something like this possible?
```
@Override
	public OpenAPI mergeOpenAPIs(
		String description, List<OpenAPI> openAPIs, String title)
```

Reply:
I updated it by including a Map<OpenAPIContext, OpenAPI> instead

Reply:
What do you think about that @4lejandrito ? Should we include that dependency? or leave it like that as it is the element returned by the OpenAPIResource interface